packages/dependency-extraction-webpack-plugin/test/__snapshots__/build.js.snap

```diff
@@ -1,5 +1,230 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`DependencyExtractionWebpackPlugin modules Webpack \`combine-assets\` should produce expected output: Asset file 'assets.php' should match snapshot 1`] = `
+"<?php return array('fileA.mjs' => array('dependencies' => array('@wordpress/blob'), 'version' => '8652d2bf4a1ea1969a6e', 'type' => 'module'), 'fileB.mjs' => array('dependencies' => array('@wordpress/token-list'), 'version' => '17d7d5b2c152592ff3a0', 'type' => 'module'));
+"
+`;
+
+exports[`DependencyExtractionWebpackPlugin modules Webpack \`combine-assets\` should produce expected output: External modules should match snapshot 1`] = `
+[
+  {
+    "externalType": "module",
+    "request": "@wordpress/blob",
+    "userRequest": "@wordpress/blob",
+  },
+  {
+    "externalType": "module",
+    "request": "@wordpress/token-list",
+    "userRequest": "@wordpress/token-list",
+  },
+]
+`;
+
+exports[`DependencyExtractionWebpackPlugin modules Webpack \`dynamic-import\` should produce expected output: Asset file 'main.asset.php' should match snapshot 1`] = `
+"<?php return array('dependencies' => array('@wordpress/blob'), 'version' => '092c2bce8c247ee11100', 'type' => 'module');
```

Comment:
A static dependency of a dynamic dependency should also be considered dynamic. 

```js
// index.js
import( './util' ).then( () => {} );
```

```js
// util.js
import { isBlobURL } from '@wordpress/blob';
```

In this case, if we mark `@wordpress/blob` as a static dependency, we will preload it when `index.js` is enqueued, which is not correct. It's true that when `index.js` finally imports `util.js`, it results in a waterfall, but that's the way modules work 🤷‍♂️